recipes/mlio/conda_build_config.yaml

```diff
@@ -0,0 +1,14 @@
+# The default deployment target used by conda-forge is 10.9 which is not
+# compatible with pybind11 due to lack of sized C++ deallocation.
+MACOSX_DEPLOYMENT_TARGET:  # [osx]
+  - 10.14                  # [osx]
+
+python:
+  - 3.6
+  - 3.7
```

Comment:
You shouldn't override like this. You should skip python 2.7 using `skip: True  # [py27]`